modules/install/pages/install.adoc

```diff
@@ -44,23 +61,165 @@ Brief blurb about uninstall.....
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# /opt/jboss-eap-installation-manager-1.1.6.Final-redhat-00001/bin/eap-8/bin/standalone.sh
+----
+
+* Navigate to the JBOSS console at http://localhost:8080
+
+[NOTE]
+--
+The jboss-eap-installation-manager is also supported on Microsoft Windows. To use this script on a Windows machine, replace the .sh with .bat in your script.
+--
 
 == Lab 2: Installing JBoss EAP using the JAR installer
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+* Install OpenJDK 17 if not already installed:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Install the GUI if not already installed on RHEL 9:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# sudo yum install java-17-openjdk -y
+----
+
+* Create a user named jboss with no shell access:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# useradd -s /sbin/nologin jboss
+----
+
+* Download the JBoss EAP zip file from https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=appplatform&downloadType=distributions[`Red Hat JBoss Enterprise Application Platform 8.0 Installer`].
+
+* Open a terminal instance and navigate to the directory containing the downloaded JBoss EAP installer file.
++
+[subs="+quotes,+macros"]
+----
+[root@localhost ~]# cd Downloads/
+[root@localhost Downloads]# ls -l jboss-eap-8.0.0-installer.jar
+-rw-r--r--. 1 root root 28141889 Apr  1 09:35 jboss-eap-8.0.0-installer.jar
+----
+
+* Run the graphical installer using the following command:
++
+[subs="+quotes,+macros"]
+----
+[root@localhost Downloads]# java -jar jboss-eap-8.0.0-installer.jar
+----
+
+* Choose your preferred language for the installer, and then click OK.
+
+*  Agree with the prompt for The EULA for RED HAT JBOSS MIDDLEWARE by selecting "I accept the terms of this license agreement", and then click Next.
+
+* Select the installation path or choose a Maven source repository for JBoss EAP, and then click Next.
++
+image::assets/images/loc.png[align="center"]
+
+* Select the components to install. Required components are disabled for deselection. Select `Next`:
++
+image::assets/images/comp.png[align="center"]
+
+* Set the admin password and select `Next`:
++
+image::assets/images/pass.png[align="center"]
+
+* Confirm the installation overview and select `Next`:
++
+image::assets/images/overview.png[align="center"]
+
+* The component installation will take around 5 mins, once completed, select `Next`:
++
+image::assets/images/complete.png[align="center"]
+
+* Select `Perform default configuration` as shown in below screenshot:
++
+image::assets/images/default.png[align="center"]
+
+* Once the progressing completes, select `Next`:
++
+image::assets/images/complete2.png[align="center"]
+
+* Finally, select `Done`:
++
+image::assets/images/complete3.png[align="center"]
+
+* Start the JBoss EAP using the start script present under `/home/jboss/EAP-8.0.0/bin/standalone.sh`
```

Comment:
i think you should install EAP under /opt as the jboss user and the resulting folders should be owned by jboss:jboss.